core/src/main/scala/org/allenai/common/StringImplicits.scala

```diff
@@ -1,115 +0,0 @@
-package org.allenai.common
```

Comment:
so this is a breaking change. Do you want to define an alias in the package object and add the deprecated annotation? I think that might be a good solution.

in package.scala:
```scala
package org.allenai

package object common {
  @deprecated("Use 'import org.allenai.common.StringUtils._' instead", since = "1.0.21")
  val StringImplicits = StringUtils
}
```